core/data/src/main/java/com/google/samples/apps/nowinandroid/core/data/repository/OfflineFirstNewsRepository.kt

```diff
@@ -45,6 +46,7 @@ private const val SYNC_BATCH_SIZE = 40
  * Reads are exclusively from local storage to support offline access.
  */
 class OfflineFirstNewsRepository @Inject constructor(
+    private val niaPreferencesDataSource: NiaPreferencesDataSource,
```

Comment:
@tunjid 
Sugestion: rename to **preferencesDataSource**, word nia is subundestend. do you think make  sense ?

Reply:
That would be out of scope for this PR. Feel free to create an issue and create a PR renaming it.